cirq-google/cirq_google/devices/grid_device.py

```diff
@@ -53,6 +53,7 @@
 _SQRT_ISWAP_GATE_FAMILY = cirq.GateFamily(cirq.SQRT_ISWAP)
 _SQRT_ISWAP_INV_GATE_FAMILY = cirq.GateFamily(cirq.SQRT_ISWAP_INV)
 _CZ_GATE_FAMILY = cirq.GateFamily(cirq.CZ)
+_CZ_POWER_GATE_FAMILY = cirq.GateFamily(cirq.CZPowGate)
```

Comment:
Consider keeping the naming convention consistent (e.g `_CZ_POW_GATE_FAMILY`) instead of `_CZ_POWER...`